src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -83,4 +83,13 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+    
+    default String getPrintableString(Printable...printables){
```

Comment:
Correct implementation of the varags method. As discussed in tutorial, there are many possible placement for this method.

Note the use of the "default" keyword, it is useful only for interface (iirc) and not for normal class methods.